src/components/ChecSlideoutPanel/Panel.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <transition
```

Comment:
Looking at https://deploy-preview-417--chec-ui.netlify.app/?path=/story/components-slideout-panel--with-stacking-panels - when I open the first panel, it transitions the width (?) then looks like it's transitioning the left margin as well? Feels slightly odd, but I can definitely live with it.

Reply:
> Looking at https://deploy-preview-417--chec-ui.netlify.app/?path=/story/components-slideout-panel--with-stacking-panels - when I open the first panel, it transitions the width (?) then looks like it's transitioning the left margin as well? Feels slightly odd, but I can definitely live with it.
> 
> If this is caused by the depth calculation logic then I think it should only be modifying margins for nested panels, rather than a single panel on its own.

I'm still trying to find a better solution to dynamically add to the panel width 😓